staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -61,6 +61,22 @@ const (
  * the metric stability policy.
  */
 var (
+	deprecatedRegisteredWatchers = compbasemetrics.NewGaugeVec(
+		&compbasemetrics.GaugeOpts{
+			Name:           "deprecated_apiserver_registered_watchers",
+			Help:           "Number of currently registered watchers for a given resources",
```

Comment:
I think maybe we can add some comment in the `Help`? Just like `Number of currently registered watchers for a given resources, will be deprecated soon`😳